loader/convert_data.go

```diff
@@ -158,10 +158,11 @@ func parseRowValues(str []byte, table *tableInfo, columnMapping *cm.Mapping) ([]
 
 	if columnMapping != nil {
 		var err error
```

Comment:
can remove this line

Reply:
removed in https://github.com/pingcap/dm/pull/269/commits/a629c67e673437a169ae8304f1c8d45562f065eb.